include/afl-fuzz.h

```diff
@@ -1180,7 +1180,6 @@ u8          havoc_mutation_probability_py(void *);
 u8          queue_get_py(void *, const u8 *);
 const char *introspection_py(void *);
 u8          queue_new_entry_py(void *, const u8 *, const u8 *);
-void        splice_optout(void *);
```

Comment:
why are you removing this?

Reply:
No function named "splice_optout" is defined. I guess what you want is `splice_optout_py` ?

Reply:
yes that would need to be `splice_optout_py` ...

Reply:
added `splice_optout_py` in next commit